content/sensu-go/6.5/files/sensu-plus-dashboard-config.json

```diff
@@ -0,0 +1,298 @@
+{
```

Comment:
Definitely going to have an updated copy of this dashboard JSON before we go live next week. And there's a pretty good chance that we'll need to maintain it over time. Let me know if that's going to be a problem.

Reply:
Perhaps we should version it somehow (via the filename)?

Reply:
I don't think it will be a problem to update this. with regard to versioning, would you expect future versions of the dashboard JSON to be incompatible with earlier versions of Sumo Logic?